### PR TITLE
Optimization for SEO

### PR DESCRIPTION
**Pull request by enescomruk72**
Meta tags were added for SEO optimization. Some front deficiencies have been fixed.

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -1,33 +1,47 @@
 <!DOCTYPE html>
 <html lang="tr">
+  <head>
+    <meta http-equiv="Content-Security-Policy" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Rythm Technologies" />
+    <title>%VITE_APP_TITLE%</title>
```
Comment (CoderMungan) [inline frontend/index.html:9]: that will be functional or ?

File: frontend/src/lib/day-js.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return dayjs(date1).isBefore(dayjs(date2));
+  }
+
+  // Bir tarihin diğerinden sonra olup olmadığını kontrol et
+  static isAfter(date1, date2) {
+    return dayjs(date1).isAfter(dayjs(date2));
+  }
+}
+
+export default DateUtils;
```
Comment (CoderMungan) [inline frontend/src/lib/day-js.js:35]: good!

File: frontend/src/pages/about/index.jsx [hunk 3]
```diff
@@ -43,7 +43,10 @@ export default function About() {
     <>
       <Helmet>
         <title>Hakkımızda</title>
-        <meta name="description" content="Rythm Tecnologies hakkımızda desc!" />
+        <meta
+          name="description"
+          content="RythmTechnologies hakkında: Teknoloji ve sanatı birleştiren özelleştirilmiş yazılım çözümleriyle, dijital dönüşüm yolculuğunuzda güvenilir partneriniz."
+        />
```
Comment (CoderMungan) [inline frontend/src/pages/about/index.jsx:49]: good!

**Review by CoderMungan — APPROVED** [review]

File: frontend/index.html (hunk 1 above)
Comment (enescomruk72) [reply frontend/index.html:9]: Yes, it will be on other pages

Comment (CoderMungan) [reply frontend/index.html:9]: i got it! gj i saw your [dcce709](https://github.com/RythmTechnologies/congenial-dollop/pull/53/commits/dcce7091b330c520a3e1a62074adfc3f47238086) last commit!

**Pull request merged**